hotline_call_times_utils.php

```diff
@@ -382,30 +449,76 @@ function addCallTime($call_time, $call_time_languages, &$error, &$message)
     $alerts = (isset($call_time['receive_call_answered_alerts']) &&
             ($call_time['receive_call_answered_alerts'] == 'on'));
     if (!$texts && !$calls && !$alerts) {
-        $error = "The call time record could not be added because no ".
+        $error = "The call time entry could not be added because no ".
                 "'received type' checkbox (texts, calls, or call ".
                 "answered alerts) was specified.";
         return false;
     }
 
-    // Add one call_times record for each language.
+    // Lock the tables.
+    if (!lockCallTimesTables($error, "The call time entry could not be added")) {
```

Comment:
I got an error here because I didn't allow table locking for the database user I had created.  I granted it the privilege and it worked fine.  I am not very familiar with tabling locking.  Is there a concern that if the script crashes in the middle, the table will not be unlocked?

Reply:
There should not be an issue; I accidentally (during testing for the development of this pull request's code) forgot to unlock the database at various times, and the database seems to auto-unlock at the end of the session. As soon as I refreshed the page or went elsewhere, the database was unlocked.